lib/src/lib.rs

```diff
@@ -4,8 +4,7 @@
 //! to provide a fully "container native" tool for using
 //! bootable container images.
 
-// See https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html
-#![deny(missing_docs)]
+#![warn(missing_docs)]
```

Comment:
Why?  What was failing?  I like having this on by default...

Reply:
As-is, it generates a const `CLAP_VERSION` that has no docstring and is marked as deprecated.  I'm trying to figure out if it's possible to just not generate that since we don't need it.

Reply:
Hmm.  Yes, or I think it would work to add an `#[allow(missing_docs)]` just on that struct if we had to.